pkgs/by-name/ve/veloren/package.nix

```diff
@@ -50,6 +50,9 @@ rustPlatform.buildRustPackage {
   ];
 
   postPatch = ''
+    # Use our Cargo.lock
```

Comment:
Maybe a comment saying we should stop doing this once upstream's lockfile builds?

Reply:
It's better to always use ours because upstream doesn't support building with stable rust.
In fact, the lockfiles were already different, probably cargo didn't complain because there were no strict constraints on the version numbers of the packages I changed.

Reply:
I'm not sure how upstream not supporting stable Rust implies we'll need a different lockfile in other cases than this one?

Reply:
The project only builds cleanly on some unstable rust version that's not packaged in Nixpkgs.
I already had to patch the lockfile to build it on rust 1.79: I don't expect I will ever be able to build with their lockfile.

Reply:
Interesting — what did you have to change that time?

Reply:
The tagged releases are typically older than the "stable" rust toolchain in Nixpkgs despite veloren being developed for unstable. So, what usually happens is that either veloren or some dependency is using an unstable feature that has been deprecated/changed/removed in the meantime and our rustc with RUSTC_BOOTSTRAP=1 can't build it anymore.

Reply:
Ahh, that makes sense.  Okay then, I think this is fine.